internal/services/devcenter/dev_center_dev_box_definition_data_source.go

```diff
@@ -112,6 +119,10 @@ func (r DevCenterDevBoxDefinitionDataSource) Read() sdk.ResourceFunc {
 						state.ImageReferenceId = pointer.From(v.Id)
 					}
 
+					if v := props.HibernateSupport; v != nil {
+						state.HibernateSupport = !strings.EqualFold(string(pointer.From(v)), string(devboxdefinitions.HibernateSupportDisabled))
+					}
```

Comment:
We should be able to do a direct comparison without casting to a string and using a case-insensitive comparison, only if this fails due to the API returning an inconsistent/incorrect casing would we use `strings.EqualFold`.

Additionally, since `pointer.From` handles the nil check for us, we can remove the condition

```suggestion
					state.HibernateSupport = pointer.From(props.HibernateSupport) == devboxdefinitions.HibernateSupportEnabled
```

Reply:
Agreed. Thanks for the suggestion!